### PR TITLE
[WIP] sig-release: Add 2021 Annual Report

### PR DESCRIPTION
**Pull request by justaugustus**
Signed-off-by: Stephen Augustus <foo@auggie.dev>

**Which issue(s) this PR fixes**:

Fixes https://github.com/kubernetes/community/issues/6377

### CODE REVIEW

File: sig-release/annual-report-2021.md [hunk 1]
```diff
@@ -39,9 +39,15 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   All of the following areas are reviewed by the Release Engineering,
```
Comment (jeremyrickard) [inline sig-release/annual-report-2021.md:42]: > All of the following areas are reviewed by the Release Engineering, subproject but we could always use more help here: 

nit: comma after subproject

File: sig-release/annual-report-2021.md [hunk 2]
```diff
@@ -57,11 +63,28 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
...
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - [Release Team](https://github.com/kubernetes/sig-release/tree/master/release-team)
+   - [Release Engineering](https://github.com/kubernetes/sig-release/tree/master/release-engineering)
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes, over the past two years, we've had contributors from the following companies (non-exhaustive, gathered from [here](https://k8s.devstats.cncf.io/d/55/company-prs-in-repository-groups?orgId=1&var-period_name=Last%202%20years&var-repogroups=SIG%20Release&var-repos=All&var-companies=All&var-countries=All)):
```
Comment (cpanato) [inline sig-release/annual-report-2021.md:71]: maybe add my current company as well `Mattermost`

Comment (puerco) [reply sig-release/annual-report-2021.md:71]: And mine (ex!)

**Comment by dims** [comment]
/assign

@justaugustus still WIP? can i start reviewing?

**Comment by justaugustus** [comment]
> @justaugustus still WIP? can i start reviewing?

@dims -- Will tag you when it's ready to roll!

**Comment by puerco** [comment]
Closing this one in favor of #6573
/close

**Pull request closed without merging**